src/core_plugins/kibana/server/tutorials/apm/saved_objects.js

```diff
@@ -0,0 +1,309 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+/* eslint max-len: 0 */
+/* eslint quotes: 0 */
```

Comment:
Instead of disabling eslint, I think the JSON should be moved to JSON-files.

Reply:
These can not move to a json file since they need javascript logic to set the apm index pattern title

Reply:
Are we talking about `staticSavedObjects`? What part of it is dynamic?